lighthouse/helpers/mongo.py

```diff
@@ -67,6 +67,27 @@ def get_positive_samples_in_source_plate(source_plate_uuid: str) -> Optional[Lis
         return None
 
 
+def get_all_samples_for_source_plate(source_plate_uuid: str) -> Optional[SampleDocs]:
+    """Attempt to get all samples for a source plate.
+
+    Arguments:
+        source_plate_uuid {str} -- The source plate UUID for which to get positive samples.
+
+    Returns:
+        {SampleDocs} -- A list of all samples on the source plate; otherwise None if they cannot be determined.
+    """
+    try:
+        samples_collection: Collection = cast(Eve, app).data.driver.db.samples
+        query = {FIELD_LH_SOURCE_PLATE_UUID: source_plate_uuid}
```

Comment:
maybe throw exception if source_plate_uuid is null, because the query {"lh_source_plate_uuid: null} is valid, but we dont want that to happen.

Reply:
It can't happen because the type hinting explicitly specifies that the value is a string that cannot be `None` but I will add the check because I think this is something that could easily slip up in future.  I can't add a test for it because mypy will complain that I'm passing a `None` value to a method that doesn't accept `None`.

Reply:
Yes, looks good